rclpy/rclpy/__init__.py

```diff
@@ -111,9 +111,9 @@ def create_node(
     namespace: str = None,
     use_global_arguments: bool = True,
     start_parameter_services: bool = True,
-    initial_parameters: List[Parameter] = None,
+    parameter_overrides: List[Parameter] = None,
     allow_undeclared_parameters: bool = False,
-    automatically_declare_initial_parameters: bool = True
+    automatically_declare_parameters_from_overrides: bool = True
```

Comment:
@jubeira how did this end up getting defaulted to `True`? Feel like we went circles around this before merging it. This does not match `rclcpp`'s default behavior and in my opinion needs to change:

- https://github.com/ros2/rclcpp/blob/0723a0a6fc2f46edcbeebf2933c7929822177d56/rclcpp/include/rclcpp/node_options.hpp#L52
- https://github.com/ros2/rclcpp/blob/0723a0a6fc2f46edcbeebf2933c7929822177d56/rclcpp/include/rclcpp/node_options.hpp#L323

Reply:
+1 to matching rclcpp

Reply:
Ah I found it, the right value is being used in the Node constructor but was left True in the `create_node` method. Fixed in 78bc9f4a6ea49165eb139e5dbfc4c31ad95a6508.

Reply:
Thanks! We must have missed it in #339.